File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int ans = 1;
        for (int i=1; i<nums.size(); i+=1){
            if (nums[i]!=nums[i-1]){
                nums[ans] = nums[i];
                ans+=1;
            }
        }

        return ans;
    }
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.go
func removeDuplicates(nums []int) int {
    i, j := 1, 1
    for ; j < len(nums); j++ {
        if nums[j] != nums[j-1] {
            nums[i] = nums[j]
            i++
        }
    }

    return i
}